in_toto/formats.py

```diff
@@ -0,0 +1,40 @@
+"""
+<Program Name>
+  formats.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  November 28, 2017.
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Format schemas for in-toto metadata, based on securesystemslib.schema.
```

Comment:
You should consider explaining how these schemata are used.  You can use the [text from securesystemslib.formats](https://github.com/secure-systems-lab/securesystemslib/blob/6d8abbc3adbfea511b76251693e1d6d894aae659/securesystemslib/formats.py#L28-L47).

Reply:
Addressed in 94986df